src/Illuminate/Database/MigrationServiceProvider.php

```diff
@@ -67,15 +93,133 @@ protected function registerCreator()
         });
     }
 
+    /**
+     * Register the given commands.
+     *
+     * @param  array  $commands
+     * @return void
+     */
+    protected function registerCommands(array $commands)
+    {
+        foreach (array_keys($commands) as $command) {
+            call_user_func_array([$this, "register{$command}Command"], []);
```

Comment:
You can use `call_user_func` function because no arguments is required.

```php
call_user_func([$this, "register{$command}Command"]);
```

Reply:
It was taken from ArtisanServiceProvider::registerCommands()
Was where any hidden sense with "people can overwrite them" ?